install/prerequisites.adoc

```diff
@@ -524,12 +524,7 @@ In the following table,
 *(L)* indicates the marked port is also used in _loopback mode_,
```

Comment:
Is there any point where loopback mode, or the fact that you're using it, is important? (If not, this is a good time to remove both the note and the annotations in the table.)

Reply:
@sdodson Can you comment on @kalexand-rh comment? I am not familiar with loopback in this context. Is loopback mode required/default? I don't see anything on how to turn loopback on/off for the install. Is this note and the [L] annotations just informational? 

This note refers to Table 3 under Required Ports:
https://docs.openshift.com/container-platform/3.9/install_config/install/prerequisites.html#required-ports

Reply:
@sdodson Before we remove the info, do you have any thoughts on why the loopback notations were added? That PR, https://github.com/openshift/openshift-docs/pull/2215, comes from a customer issue, so I want to see if the information could be helpful.
https://access.redhat.com/support/cases/#/case/01544754 (In Japanese)

Reply:
I read the bug but wasn't quite sure of the purpose, I think the customer made an assumption that the ports are only necessary when operating multiple masters because the section is "Masters to Masters". I think that's an incorrect assumption to make, master-A -> master-A is still "Master to Master". I think as it is now it's a case of too much information being confusing and it should be removed.
@nak3 what do you think about removing these notes about loopback here?

Reply:
Yes, I agree. It should be removed. I checked bz#1287949 and it looks like my first comment was wrong. I'm sorry :fearful: